test/optics/radius_newton_ring_from_wavelength_order_radius_curvature_refractive_index_test.py

```diff
@@ -0,0 +1,59 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from sympy import pi
+from symplyphysics import (units, SI, convert_to, Quantity, errors)
+from symplyphysics.laws.optics import radius_newton_ring_from_wavelength_order_radius_curvature_refractive_index as radius_law
+
+# Description
+## Let the wavelength be 589 nanometer, the order of interference is 6,
+## radius of curvature is 4.99 meter and the refractive index is 1.
+## Then the radius of the dark Newtons's ring will be 4.2 millimeter.
+## https://uchi.ru/otvety/questions/diametr-shestogo-temnogo-koltsa-nyutona-okazalsya-ravnim-d6-84mm-ustanovka-dlya-polucheni?utm_referrer=https%3a%2f%2fwww.google.ru%2f
```

Comment:
Link is broken.

Reply:
> Link is broken.

I just clicked on this link. Everything works.
![image](https://github.com/blackyblack/symplyphysics/assets/100017508/42e01e2c-050f-4fe8-a3fa-194b0dd2691e)